cmd/protoc-gen-gorums/dev/strictordering.go

```diff
@@ -0,0 +1,165 @@
+package dev
+
+import (
+	"context"
+	"math"
+	"math/rand"
+	"sync"
+	atomic "sync/atomic"
+	"time"
+
+	"github.com/relab/gorums"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/backoff"
+	"google.golang.org/grpc/codes"
+	status "google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/anypb"
+)
+
+type strictOrderingResult struct {
+	nid   uint32
+	reply *anypb.Any
+	err   error
+}
+
+type requestHandler func(*gorums.Message) *gorums.Message
+
+type strictOrderingManager struct {
```

Comment:
Don't think this struct's name is representative for its content. What about `receiveBuffer` or `receiveQueue`?

Also, I propose to augment its API with functions for safely accessing the map, e.g. Load/Store/Delete. With this API, you could either forward to the corresponding `sync.Map` API or encapsulate snippets like this:
```go
	s.recvQMut.RLock()
	if c, ok := s.recvQ[resp.GetID()]; ok {
		c <- &strictOrderingResult{nid: s.node.ID(), reply: resp.GetData(), err: nil}
	}
	s.recvQMut.RUnlock()
```